examples/pdbbind/pdbbind_atomic_conv.py

```diff
@@ -0,0 +1,222 @@
+    frag1_outputs = self.in_layers[0].out_tensor
+    frag2_outputs = self.in_layers[1].out_tensor
+    complex_outputs = self.in_layers[2].out_tensor
+
+    frag1_energy = tf.reduce_sum(frag1_outputs, 0)
+    frag2_energy = tf.reduce_sum(frag2_outputs, 0)
+    complex_energy = tf.reduce_sum(complex_outputs, 0)
+    binding_energy = complex_energy - frag1_energy - frag2_energy
+    self.out_tensor = tf.expand_dims(binding_energy, axis=1)
+    return self.out_tensor
+
+
+transformers = [
+    dc.trans.NormalizationTransformer(transform_y=True, dataset=train_dataset)
+]
+
+for transformer in transformers:
+  train_dataset = transformer.transform(train_dataset)
+  test_dataset = transformer.transform(test_dataset)
+
+frag1_num_atoms = 140
+frag2_num_atoms = 821
+complex_num_atoms = 908
+max_num_neighbors = 12
+neighbor_cutoff = 12.0
+batch_size = 300
+
+at = [1., 6, 7., 8., 9., 11., 12., 15., 16., 17., 20., 25., 30., 35., 53.]
+radial = [[12.0], [0.0, 4.0, 8.0], [4.0]]
+rp = [x for x in itertools.product(*radial)]
+layer_sizes = [32, 32, 16, 1]
+dropouts = [0., 0., 0.]
+penalty = 0.
+
+frag1_X = Feature(shape=(batch_size, frag1_num_atoms, 3))
+frag1_nbrs = Feature(shape=(batch_size, frag1_num_atoms, max_num_neighbors))
+frag1_nbrs_z = Feature(shape=(batch_size, frag1_num_atoms, max_num_neighbors))
+
+frag2_X = Feature(shape=(batch_size, frag2_num_atoms, 3))
+frag2_nbrs = Feature(shape=(batch_size, frag2_num_atoms, max_num_neighbors))
+frag2_nbrs_z = Feature(shape=(batch_size, frag2_num_atoms, max_num_neighbors))
+
+complex_X = Feature(shape=(batch_size, complex_num_atoms, 3))
+complex_nbrs = Feature(shape=(batch_size, complex_num_atoms, max_num_neighbors))
+complex_nbrs_z = Feature(shape=(batch_size, complex_num_atoms,
+                                max_num_neighbors))
+
+frag1_conv = AtomicConvolution(
+    atom_types=at,
+    radial_params=rp,
+    boxsize=None,
+    in_layers=[frag1_X, frag1_nbrs, frag1_nbrs_z])
+
+frag2_conv = AtomicConvolution(
+    atom_types=at,
+    radial_params=rp,
+    boxsize=None,
+    in_layers=[frag2_X, frag2_nbrs, frag2_nbrs_z])
+
+complex_conv = AtomicConvolution(
+    atom_types=at,
+    radial_params=rp,
+    boxsize=None,
+    in_layers=[complex_X, complex_nbrs, complex_nbrs_z])
+
+frag1_conv = Transpose(out_shape=[2, 1, 0], in_layers=[frag1_conv])
+frag2_conv = Transpose(out_shape=[2, 1, 0], in_layers=[frag2_conv])
+complex_conv = Transpose(out_shape=[2, 1, 0], in_layers=[complex_conv])
+
+for layer_size in layer_sizes:
+  frag1_conv = Dense(
+      out_channels=layer_size,
+      activation_fn=tf.nn.relu,
+      time_series=True,
+      in_layers=[frag1_conv])
+  frag2_conv = frag1_conv.shared(in_layers=[frag2_conv])
+  complex_conv = frag1_conv.shared(in_layers=[complex_conv])
+
+score = AtomicConvScore(in_layers=[frag1_conv, frag2_conv, complex_conv])
+
+label = Label(shape=(None, 1))
+loss = L2LossLayer(in_layers=[score, label])
+
+
+def feed_dict_generator(dataset, batch_size, epochs=1):
```

Comment:
This is pretty complicated :). Let's brainstorm ways in future PRs to make constructing feed-dicts easier for complex models.